analytics/policy/services/business_licence_client.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+import urllib3
+from models.listing import Listing
+from urllib3.exceptions import HTTPError
+
+
+class BusinessLicenceClient:
+    BASE_URL = "https://opendata.vancouver.ca/api/explore/v2.1/catalog/datasets/business-licences/records"
+
+    def __init__(self) -> None:
+        self.session = urllib3.PoolManager()
+
+    def _filter_by_licence_number(self, licence_number: str) -> dict:
+        # Omit licence_number validity check since it should be checked during policy evaluation
+        return {"where": f'licencenumber="{licence_number}"'}
+
+    def _filter_by_short_term_rental_business(self) -> dict:
+        return {"where": 'businesstype="Short-Term Rental"'}
+
+    def _licence_status_query(self) -> dict:
+        return {"select": "status"}
+
+    def _make_request(self, fields: dict) -> dict:
+        try:
+            response = self.session.request(
+                method="GET", url=self.BASE_URL, fields=fields
+            )
+            return json.loads(response.data)
+        except HTTPError as e:
+            print(f"Error fetching data: {e}")
+            raise e
+        except json.JSONDecodeError as e:
+            print(f"Error decoding data: {e}")
+            raise e
+
+    def _process_licence_status_results(self, response_data: dict) -> str | list[str]:
+        total_count = response_data["total_count"]
+        results = response_data["results"]
+        licence_status = [result["status"] for result in results]
+
+        if total_count > 1:
+            print(
+                f"Received {total_count} counts of licence number with statuses: {licence_status}"
+            )
+        return licence_status
+
+    def _merge_query_parameters(self, *parameters: dict) -> dict:
+        merged_query_parameter = {}
+        for parameter in parameters:
+            for key, value in parameter.items():
+                if key in merged_query_parameter:
+                    merged_query_parameter[key] = " and ".join(
+                        [merged_query_parameter[key], value]
+                    )
+                else:
+                    merged_query_parameter[key] = value
+        return merged_query_parameter
+
+    def get_licence_status(self, licence_number: str) -> list[str]:
```

Comment:
what is the expected output format of this method? document it here. it is not clear to me what the response is. 
what is the list[str]? list of statuses? if so, add a quick method doc here.
also indicate what are the possible statuses. that way the consumer of this method knows how to utilize it.

Reply:
It will return a list of status (str)
I will add comments to clarify this